database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -70,11 +70,10 @@
  * @param <VH> The ViewHolder class that contains the Views in the layout that is shown for each object.
  */
 public abstract class FirebaseRecyclerAdapter<T, VH extends RecyclerView.ViewHolder> extends RecyclerView.Adapter<VH> {
-
-    Class<T> mModelClass;
+    protected Class<T> mModelClass;
     protected int mModelLayout;
-    Class<VH> mViewHolderClass;
-    FirebaseArray mSnapshots;
+    protected Class<VH> mViewHolderClass;
+    protected FirebaseArray mSnapshots;
```

Comment:
This does leave me wondering how multiple ViewHolders and models could be used. Maybe I'll mess around with that today.